scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -1335,6 +1335,12 @@ def start_service(self, service, idx):
     cmd = f"cp -v {self.local_path}/motr/sysconfig/{self.machine_id}/motr /etc/sysconfig/"
     execute_command(self, cmd)
 
+    if service == "ioservice" or service == "confd":
```

Comment:
This check is not required and also move the code to below 1346 line, need to have a valid index before proceeding.